Makefile

```
@@ -1,5 +1,5 @@
 DOCKER       = docker
-HUGO_VERSION = 0.59.1
+HUGO_VERSION = $(shell cat netlify.toml | grep HUGO_VERSION|cut -d' ' -f3|tr -d '"')
```

@Colstuwjx how about this:
```suggestion
HUGO_VERSION = $(shell grep ^HUGO_VERSION netlify.toml | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
```

I think it will be more robust.

---

Yes! It's better than the original one, and it could avoid sth like `HUGO_VERSION =   0.59.1` ( carried too many spaces case).

But this command execute result was `0.59.1%` if we just have one line `HUGO_VERSION` set, it has ending char on my MBP, I am wondering does it matters ( the check script passed ).

---

@Colstuwjx is your prompt set to `%` maybe?

---

Hi @sftim ,

I have been checked this issue, and it was caused by [on-my-zsh display](https://github.com/ohmyzsh/ohmyzsh/issues/7832#issuecomment-489450121)

I have been updated to your suggested cmd since we are using ubuntu image for netlify build from our CI, and it works fine on Linux ( MacOS also works well, just display issue ).

Thanks for your suggested cmd again :)